src/SlashCommands.tsx

```diff
@@ -1026,6 +1028,27 @@ export const Commands = [
         },
         category: CommandCategories.actions,
     }),
+    new Command({
+        command: "confetti",
+        description: _td("Sends the given message with confetti"),
+        args: '<message>',
+        runFn: function(roomId, args) {
+            return success((async () => {
+                const cli = MatrixClientPeg.get();
+                const userId = cli.getUserId();
+                const userName = userId.slice(1).split(":").slice(0, 1);
+                const isChatEffectsDisabled = SettingsStore.getValue('dontShowChatEffects');
+                if (!args || isChatEffectsDisabled) {
+                    args = '*' + userName + _td(' sends confetti');
```

Comment:
this is poor for translation, should be more like `_t("* %(userName)s sends confetti")`

Reply:
in this case the args will be coonverted to span component and the text would be in its props, right? how can it be displayed as string in sendHtmlMessage(roomId, args)? 
as it is now it sends an empty message.

Reply:
`_t()` with only 2 args returns a string, not a span

Reply:
![Bildschirmfoto 2020-08-25 um 10 52 12](https://user-images.githubusercontent.com/34133773/91153892-1abea600-e6c1-11ea-84b8-dbc4826313c1.jpg)
![Bildschirmfoto 2020-08-25 um 10 51 14](https://user-images.githubusercontent.com/34133773/91153907-20b48700-e6c1-11ea-82aa-326324b0cbbb.jpg)
this is what i get when logging args

Reply:
Can you commit your latest code so that I can test it